src/controllers/event/add.ts

```diff
@@ -118,15 +148,41 @@ const createEvent: Interfaces.Controller.Async = async (req, res, next) => {
       module: {
         connect: { id: moduleId },
       },
-      organizers: {
-        connect: organizersUsernames,
-      },
-      managers: {
-        connect: managersUsernames,
-      },
     },
   });
 
+  if (organizers) {
+    try {
+      organizers?.map(async (id: any) => {
```

Comment:
remove the `?` operator before map as you are already checking for the presence of organizers 

If possible give `id` a more specific type than `any`, and try to not use `any` as much as possible

Reply:
I have removed the `?` operator and gave the type `string` to the `id`.